src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -77,6 +78,7 @@ export const ConsultationDetails = (props: any) => {
     useState(false);
   const [openDischargeDialog, setOpenDischargeDialog] = useState(false);
   const [showAutomatedRounds, setShowAutomatedRounds] = useState(true);
+  const [showPatientNotesPopup, setShowPatientNotesPopup] = useState(false);
```

Comment:
Sometime later we can use React Context for this state so that we can show doctor notes across multiple pages maybe. Just a thought, can be implemented later after discussions.